lib/aims/realtime_util.py

```diff
@@ -89,6 +91,51 @@ def _pickle_filename(level_qc):
 
     return pickle_qc_file
 
+def delete_channel_id_from_pickle(level_qc, channel_id):
```

Comment:
> How often do you think you'll need to do this?

not often. but their service is not the most reliable one, and they tend to change things often. So definitely could be useful